app/gui2/shared/yjsModel.ts

```diff
@@ -28,7 +29,13 @@ export function visMetadataEquals(
 ) {
   return (
     (a == null && b == null) ||
-    (a != null && b != null && a.visible === b.visible && visIdentifierEquals(a, b))
+    (a != null &&
+      b != null &&
+      a.visible === b.visible &&
+      ((a.identifier == null && b.identifier == null) ||
+        (a.identifier != null &&
+          b.identifier != null &&
+          visIdentifierEquals(a.identifier, b.identifier))))
```

Comment:
Please split this expression, maybe to some `if`. Or perhaps `visIdentifierEquals` should accept nulls?
You can also use `compareOpt`.

Reply:
i'll probably go with `visIdentifierEquals` accepting `null`s - `shared/` does not have access to `util`.